README.md

```diff
@@ -1,85 +1,46 @@
-Terraform Provider
-==================
+<a href="https://terraform.io">
+  <img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" title="Terraform" align="right" height="50" />
+</a>
 
-- Website: https://www.terraform.io
-- [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
-- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+# Terraform Provider for Grafana
 
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
+- Terraform website: https://terraform.io
+- Grafana website: https://grafana.com
+- Provider Documentation: https://www.terraform.io/docs/providers/grafana/index.html
```

Comment:
FWIW this redirects you to `https://registry.terraform.io/providers/grafana/grafana/latest/docs`, so we might as well link that directly:

```suggestion
- Provider Documentation: https://registry.terraform.io/providers/grafana/grafana/latest/docs
```